aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Card.java

```diff
@@ -0,0 +1,110 @@
+package io.sunshower.zephyr.ui.components;
+
+import com.vaadin.flow.component.ClickEvent;
+import com.vaadin.flow.component.ClickNotifier;
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.HtmlContainer;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.dependency.CssImport;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.html.Footer;
+import com.vaadin.flow.component.html.Header;
+import com.vaadin.flow.component.html.Section;
+import lombok.NonNull;
+import lombok.val;
+
+@Tag("aire-card")
+@JsModule("./aire/ui/components/card.ts")
+@CssImport("./styles/aire/ui/components/card.css")
+public class Card extends HtmlContainer implements ClickNotifier<Card> {
+
+  public static final String SLOT = "slot";
```

Comment:
We use these strings a lot throughout -- worth pulling into a separate set of constants?

Reply:
I hear what you're saying, but after some thought, I think that colocating styles has some disadvantages:

1. It removes the association between a style and its host component
2. Changing the style could break multiple components